app/app.rb

```diff
@@ -89,17 +89,17 @@
   set :views, 'app/views'
 end
 
-get '/' do
-  haml :maintenance, layout: false
-end
-
 # get '/' do
-#   redirect '/home'
+# haml :maintenance, layout: false
 # end
 
-# get '/home' do
-#   send_file 'static/home.html'
-# end
+get '/' do
+   redirect '/home'
+end
+
+ get '/home' do
+   send_file 'static/home.html'
+ end
```

Comment:
why!? haha

how about this...
```suggestion
get '/' do
   redirect '/hell'
end

 get '/home' do
   send_file 'static/hotel.html'
 end
```

Reply:
that ok, it's correct